vllm/entrypoints/api_server.py

```diff
@@ -107,6 +110,13 @@ async def stream_results() -> AsyncGenerator[bytes, None]:
         engine_args, usage_context=UsageContext.API_SERVER)
 
     app.root_path = args.root_path
+
+    logger.info("Available endpoints are:")
+    for route in app.routes:
+        methods = ', '.join(route.methods) if hasattr(
+            route, 'methods') else 'No methods'
```

Comment:
What routes would have no method?

Reply:
fixed in [b64797c](https://github.com/vllm-project/vllm/pull/6195/commits/b64797cbb5d45dc2588dc1071b0b4d63dd830350). maybe this is for edge cases? chatgpt adds it, and i think it is fine to add the handling logic :)

Reply:
I'm mostly just curious lol
Then maybe we could just use "N/A" for concise.

Reply:
p.s. Filtering out is also fine.